src/Microsoft.AspNet.Razor/Properties/AssemblyInfo.cs

```diff
@@ -5,5 +5,4 @@
 using System.Runtime.CompilerServices;
 
 [assembly: InternalsVisibleTo("Microsoft.AspNet.Razor.Test")]
-[assembly: InternalsVisibleTo("Microsoft.AspNet.Razor.Runtime.Test")]
```

Comment:
It says:

>  `InternalsVisibleTo` is used only to allow a unit test to test internal types and members of its runtime assembly.

Because there is a 1:1 mapping between a unit test assembly and a runtime assembly<sup>[citation available](https://github.com/aspnet/Home/wiki/Engineering-guidelines#assembly-naming)</sup>, the IVT restriction is the logical conclusion of those two guidelines.

Let's please table this discussion now.